examples/Latency/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+# CMake file for DISTRHO Plugins #
+# ------------------------------ #
+
+dpf_add_plugin(d_latency
+  TARGETS ladspa dssi lv2 vst
```

Comment:
latency has no UI, so dssi is pointless here (the exposed plugin is not a synth, makes no use of dssi stuff)